README.md

```diff
@@ -183,6 +183,20 @@ _[:inverse_name]_ Specifies the name of parent object used by the association. T
 Example:
 `cache_has_one :configuration, :embed => true`
 
+#### cache_belongs_to
+
+Options:  
```

Comment:
Wow, I didn't realized how much we let users override from the options until now.  Most of these options can also be overriden on other configuration dsl methods, but are undocumented, so I'm not sure if they were all intentionally exposed.  I don't think we should allow them to be overridden at all in the future, so I don't think we should start documenting them now.